packages/react/src/ProgressBar/ProgressBar.stories.tsx

```diff
@@ -12,13 +19,21 @@ export default {
 export const Default = () => <ProgressBar aria-label="Upload test.png" />
 
 export const Playground = ({sections, ...args}: ProgressBarProps & {sections: number}) => {
+  const [sectionColors, setSectionColors] = React.useState(sectionColorsDefault)
+
+  useEffect(() => {
+    if (args.bg && args.bg !== '') {
+      setSectionColors([args.bg, ...sectionColorsDefault])
+    }
+  }, [args.bg])
```

Comment:
Didn't really understand what this block of code does, can you help me?

Reply:
Needed to avoid error: "To many re-renders"